### ISSUE TITLE
Not able to reproduce lpRec example

### ISSUE THREAD

**Issue by venkat-purdue** [tomopy/tomopy#363]
I have tomopy version 1.2.1 on a Ubuntu 16.04 machine. I also installed Lprec from https://github.com/math-vrn/lprec. 

When I try running the example https://tomopy.readthedocs.io/en/latest/ipynb/lprec.html, the output of the reconstruction is not what is shown in the result and instead an array with values 1.e-06. 

Any advice on what could be going on ?

**Comment by math-vrn** [comment]
@venkat-purdue , could you please first check that cupy (https://cupy.chainer.org/)) is installed and then run "python ./tests/run_test.py" from lprec directory to perform all unit tests?

**Comment by venkat-purdue** [comment]
@math-vrn 

cupy.__version__
'4.4.1'

python tests/run_test.py 

test_adj (__main__.TestLpMethods) ... [0.95586085, 0.0011777706812102892]
ok
test_adj_many (__main__.TestLpMethods) ... [0.002050212702084428, 0.005194131, 0.00052730384]
ok
test_adj_ptr (__main__.TestLpMethods) ... 0.001416860182196358
ok
test_foam (__main__.TestLpMethods) ... [602.86017, 1181.3875, 1139.512, 444.35748, 926.6202]
ok
test_gpus_many (__main__.TestLpMethods) ... 381.52924
ok
test_gpus_many_map (__main__.TestLpMethods) ... 678.3957
ok
test_imp (__main__.TestLpMethods) ... [1993.7755, 3555.668, 3454.9526, 2933.0613, 3355.1602]
FAIL
test_lpmethods (__main__.TestLpMethods) ... [222.42381, 348.76956, 347.48557, 328.2814, 324.60397]
ok

======================================================================
FAIL: test_imp (__main__.TestLpMethods)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "tests/run_test.py", line 43, in test_imp
    (norm2-3454.9438 < dif) & (norm3-2933.0613 < dif) & (norm4-3355.1555 < dif))
AssertionError: False is not true

----------------------------------------------------------------------
Ran 8 tests in 95.167s

FAILED (failures=1)

**Comment by math-vrn** [comment]
Are you using tomopy from conda-forge or you cloned it from github? Conda-forge now is using the old lprec wrapper version. Could you check that your tomopy/tomopy/recon/wrapper.py is the same as in the master branch?

**Comment by venkat-purdue** [comment]
I am using tomopy from conda but the version is 1.2.1. Do I still need to build from source using the current master branch ?

**Comment by dgursoy** [comment]
Yes and sorry about not being prompt on versioning between the releases. Typically the version is increased just before the release, but we should increase it just after the release instead. Please try it from the GitHub master branch.

**Comment by math-vrn** [comment]
@dgursoy Thanks!

**Comment by venkat-purdue** [comment]
I see. 

I am getting an mkl error when I try to install the master.

gcc -pthread  -DNDEBUG -g -fwrapv -O3 -fPIC -Wsign-compare -Wall -Wstrict-prototypes  -I/software/tomopy/include -I/anaconda2/include -I/anaconda2/include/python2.7 -DUSE_MKL -std=c99     -c -o gridrec.o ../src/gridrec.c
../src/gridrec.c:57:21: fatal error: mkl.h: No such file or directory

**Comment by math-vrn** [comment]
Try conda install mkl-include
